usr/share/rear/verify/BAREOS/default/050_check_requirements.sh

```diff
@@ -18,17 +20,46 @@ if [ "$BEXTRACT_DEVICE" -o "$BEXTRACT_VOLUME" ]; then
 
 else
 
-   ### Bareos support using bconsole
-   [ -x /usr/sbin/bareos-fd ]
-   StopIfError "Bareos executable (bareos-fd) missing or not executable"
+    ### Bareos support using bconsole
+
+    if ! has_binary bareos-fd; then
```

Comment:
Please don't check this in `verify` but in `prep`. In the rescue environment we must rely on the stuff prepared during `mkrescue` and for Bareos creating a rescue image without Bareos in it should not be possible.

Reply:
@schlomo good catch - thanks.
@joergsteffens Can you update your PR as such?